doc/DeveloperGuide.md

```diff
@@ -90,6 +90,36 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename tag
+
+**MSS**
+
+1. User requests to rename tag
+2. AddressBook shows a list of persons with tag
+3. AddressBook requests user to key in tag to change
```

Comment:
Remember the definition of a use case. It doesn't make sense to have systems/users in more than 1 consecutive steps.